src/v2/controllers/FormController.js

```diff
@@ -166,8 +166,18 @@ export default Controller.extend({
     }
 
     idx.proceed(formName, model.toJSON())
-      .then(this.handleIdxSuccess.bind(this))
-      .catch(error => {
+      .then((resp) => {
+        const onSuccess = this.handleIdxSuccess.bind(this);
+
+        if (formName === FORMS.ENROLL_PROFILE) {
+          // call registration (aka enroll profile) hook
+          this.settings.postSubmit(resp, onSuccess, (error) => {
+            throw error;
```

Comment:
Can we try to show this error similar to how we did it in V1  or trigger model.error ?

Reply:
https://github.com/okta/okta-signin-widget/pull/1747/commits/a5231462b7dd1777ff44f8b20e10f482283fc388